### DISCUSSION TITLE
Screenshot

### DISCUSSION THREAD

**Discussion by Azharlashari** [ReVanced/revanced-discussions#332] [General]
I am a student i watch lectures on youtube and screenshot important things so i need screenshot button inbuilt in revanced it will help me alot because taking screenshot with button is little difficult.

**Comment by TheJeterLP** [comment]
Just use the feature android already has?

**Reply by Azharlashari** [reply]
Which feature??

**Reply by TheJeterLP** [reply]
Every phone has the ability to press two buttons to create a screenshot. Or you can make a screenshot from the quicksettings menu.

**Reply by oSumAtrIX** [reply]
> because taking screenshot with button is little difficult.

**Reply by TheJeterLP** [reply]
> > because taking screenshot with button is little difficult.

but its not?

**Comment by alok-old** [comment]
I don't think it will be reasonable to implement it to YouTube, I don't know if I am allowed to give suggestion of another app but you can use "Tap, Tap" app and assign screenshot gesture to it. It will help if you take so many screenshots.

**Reply by Azharlashari** [reply]
Yes i tried many apps even my android has default feature floating screenshot but the problem is delay which cause screenshot after 0.5 sec and that time video content or text missed so  i have to repeat video and pause to take screen shot implement inbuilt screenshot can take instant screenshot without delay if developers work on it i think it will be useful if i can i would do it myself but i cant because I'm not developer. It will be my dream feature because i took so many screenshot i have 5 gb data of screenshots.

**Reply by TheJeterLP** [reply]
For now this is not planned.

**Reply by venkat2305** [reply]
When implementation of features from vanced is done, please consider this. It will be very useful
